ballerina-tests/tests/40_bidirectional_streaming_negative_test_service.bal

```diff
@@ -13,18 +13,19 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-import ballerina/log;
 
-listener Listener ep40 = new (9140);
+import ballerina/grpc;import ballerina/log;
```

Comment:
Need a new line.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/27911584b1b415028a5454b5888d76ca4f315b2d